.github/workflows/deploy.yml

```diff
@@ -65,47 +61,45 @@ jobs:
              fi
 
              if [ "${{github.event.inputs.environment }}" == "Development" ]
              then
-                 echo ::set-output name=ARM_ACCESS_KEY::$(echo "${{ secrets.DEV_ARM_ACCESS_KEY  }}" )
-                 echo ::set-output name=RAILS_ENV::$(echo "rolling" )
-                 echo ::set-output name=RAILS_MASTER_KEY::$(echo "${{ secrets.RAILS_MASTER_KEY_ROLLING }}" )
-                 echo ::set-output name=HTTP_PASSWORD::$(echo "${{ secrets.HTTPAUTH_PASSWORD }}" )
-                 echo ::set-output name=HTTP_USERNAME::$(echo "${{ secrets.HTTPAUTH_USERNAME }}" )
                  echo ::set-output name=control::$(echo "dev" )
                  echo ::set-output name=healthcheck::$(echo "${{env.PAAS_APPLICATION_NAME}}-dev" )
                  echo ::set-output name=key::"app.dev.terraform"
                  echo ::set-output name=docker_image::${{env.DOCKERHUB_REPOSITORY}}:sha-${{ steps.sha.outputs.short}}
+                 echo "VAULT=s146d01-kv"       >> $GITHUB_ENV
              fi
 
              if [ "${{github.event.inputs.environment }}" == "Test" ]
              then
-                 echo ::set-output name=ARM_ACCESS_KEY::$(echo "${{ secrets.TEST_ARM_ACCESS_KEY  }}" )
-                 echo ::set-output name=RAILS_ENV::$(echo "preprod" )
-                 echo ::set-output name=RAILS_MASTER_KEY::$(echo "${{ secrets.RAILS_MASTER_KEY_PREPROD }}" )
-                 echo ::set-output name=HTTP_PASSWORD::$(echo "${{ secrets.HTTPAUTH_PASSWORD }}" )
-                 echo ::set-output name=HTTP_USERNAME::$(echo "${{ secrets.HTTPAUTH_USERNAME }}" )
                  echo ::set-output name=control::$(echo "test" )
                  echo ::set-output name=healthcheck::$(echo "${{env.PAAS_APPLICATION_NAME}}-test" )
                  echo ::set-output name=key::"app.test.terraform"
                  echo ::set-output name=docker_image::${{env.DOCKERHUB_REPOSITORY}}:sha-${{ steps.sha.outputs.short}}
+                 echo "VAULT=s146t01-kv"       >> $GITHUB_ENV
              fi
 
              if [ "${{github.event.inputs.environment }}" == "Production" ]
              then
-                 echo ::set-output name=ARM_ACCESS_KEY::$(echo "${{ secrets.PROD_ARM_ACCESS_KEY  }}" )
-                 echo ::set-output name=RAILS_ENV::$(echo "production" )
-                 echo ::set-output name=RAILS_MASTER_KEY::$(echo "${{ secrets.RAILS_MASTER_KEY_PRODUCTION }}" )
-                 echo ::set-output name=HTTP_PASSWORD::$(echo "" )
-                 echo ::set-output name=HTTP_USERNAME::$(echo "" )
-                 echo ::set-output name=control::$(echo "production" )
                  echo ::set-output name=healthcheck::$(echo "${{env.PAAS_APPLICATION_NAME}}-prod" )
                  echo ::set-output name=key::"app.production.terraform"
                  echo ::set-output name=docker_image::${{env.DOCKERHUB_REPOSITORY}}:sha-${{ steps.sha.outputs.short}}
+                 echo "VAULT=s146p01-kv"       >> $GITHUB_ENV
```

Comment:
is this required? if you are using environment scoped secrets, 
you could just assign them as secrets and `use ${{ secrets.VAULT }}` in the keyvault step below.

Reply:
True. Never thought of that